src/components/InputWithLabel.jsx

```diff
@@ -0,0 +1,30 @@
+import "./inputWithLabel.css"
+function InputWithLabel(props) {
+    const type = props.Type;
+    if (type == "input") {
+        return (
+            <div className="input">
+                <label className="label" >{props.Labelvalue}</label>
+                <form >
+                    <input type="text" placeholder={"Enter " + props.Labelvalue}/>
```

Comment:
It would be better if we pass a custom placeholder prop instead of use the label as the placeholder